### ISSUE TITLE
Support muCommander

### ISSUE THREAD

**Issue by jonelo** [jonelo/jacksum-fbi-windows#3]
muCommander supports custom commands stored in a file called commands.xml, it is saved under %USERPROFILE%\.mucommander on Windows. See also https://github.com/mucommander/mucommander/wiki/Preference-Files

So it is a candidate in order to expand the list of supported file managers for the Jacksum File Browser Integration for Windows installer.

**Comment by jonelo** [comment]
As soon as the File Browser Integration script have run, the functions can be accessed with the following commands in %USERPROFILE%\.mucommander\commands.xml; user Johann, locale German in the example below:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!--
  Jacksum muCommander integration/configuration file

  If you want to have HashGarten/Jacksum (https://jacksum.net)
  integrated in muCommander (https://www.mucommander.com),
  copy this file (commands.xml) to the .mucommander directory,
  see also https://github.com/mucommander/mucommander/wiki/Preference-Files#locating-the-customisation-files
-->
<commands>
  <command alias="Jacksum - 1) Berechne Hashwerte"            value="cmd /c start &quot;&quot; &quot;C:\\Users\\Johann\\AppData\\Roaming\\Microsoft\\Windows\\SendTo\\Jacksum\ -\ 1)\ Berechne\ Hashwerte.lnk&quot; &quot;$f&quot;" />
  <command alias="Jacksum - 2) Überprüfe die Datenintegrität" value="cmd /c start &quot;&quot; &quot;C:\\Users\\Johann\\AppData\\Roaming\\Microsoft\\Windows\\SendTo\\Jacksum\ -\ 2)\ Überprüfe\ die\ Datenintegrität&quot; &quot;$f&quot;" />
  <command alias="Jacksum - 3) Benutzerdefinierte Ausgabe"    value="cmd /c start &quot;&quot; &quot;C:\\Users\\Johann\\AppData\\Roaming\\Microsoft\\Windows\\SendTo\\Jacksum\ -\ 3)\ Benutzerdefinierte Ausgabe.lnk&quot; &quot;$f&quot;" />
  <command alias="Jacksum - 4) Bearbeite Batch"               value="cmd /c start &quot;&quot; &quot;C:\\Users\\Johann\\AppData\\Roaming\\Microsoft\\Windows\\SendTo\\Jacksum\ -\ 4)\ Bearbeite\ Batch.lnk&quot;" />
  <command alias="Jacksum - 5) Hilfe"                         value="cmd /c start &quot;&quot; &quot;C:\\Users\\Johann\\AppData\\Roaming\\Microsoft\\Windows\\SendTo\\Jacksum\ -\ 5)\ Hilfe.lnk&quot;" />
</commands>
```

Limit of the muCommander 1.3 API: only one file or directory can be transferred to HashGarten (with $f), but this is not a big problem, because both muCommander and HashGarten support drag & drop to transfer file/directory-paths to and from the tools.

**Comment by jonelo** [comment]
Implemented in version 2.7.0. See also commit b88e50df238a86f474b733f82ee97faa58e688a3

After the installation a suitable and localized commands.xml is located in the installation directory. The file contains further instructions on how to to install it in mucommander.
